InvoiceMaker/InvoiceMaker/FormModels/EditWorkDone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using InvoiceMaker.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.FormModels
+{
+    public class EditWorkDone
+    {
+        public Client Client { get; set; }
+        public WorkType WorkType { get; set; }
```

Comment:
This form model looks incomplete. Are you going to allow the user to edit the `StartedOn` and/or `EndedOn` values?